src/main/kotlin/com/learnspigot/bot/command/StartvoteCommand.kt

```diff
@@ -0,0 +1,123 @@
+                        color = EMBED_COLOR
+                    }).queue()
+                    val tutorialEmojis = mutableListOf("0️⃣", "1️⃣", "2️⃣", "3️⃣", "4️⃣", "5️⃣")
+                    val projectEmojis =
+                        mutableListOf("0️⃣", "1️⃣", "2️⃣", "3️⃣", "4️⃣", "5️⃣", "6️⃣", "7️⃣", "8️⃣", "9️⃣", "🔟")
+                    val type = bot.getThreadChannelById(it.channel!!.id)!!.appliedTags[0].name
+                    var message: Message? = null
+                    bot.getTextChannelById(System.getenv("VOTE_CHANNEL_ID"))!!
+                        .sendMessageEmbeds(voteEmbed(type, it, false)).queue { m ->
+                            message = m
+                            if (type == "Tutorial") tutorialEmojis.forEach { e ->
+                                m.addReaction(Emoji.fromUnicode(e)).queue()
+                            }
+                            else projectEmojis.forEach { e -> m.addReaction(Emoji.fromUnicode(e)).queue() }
+                        }
+                    Timer().schedule(86400000) {
+                        var sum = 0
+                        val usersReacted = mutableListOf<String>()
+                        val usersReaction = mutableListOf<Int>()
+                        if (type == "Tutorial") {
+                            for (emo in tutorialEmojis.indices.reversed()) {
+                                for (user in bot.getTextChannelById(System.getenv("VOTE_CHANNEL_ID"))!!
+                                    .retrieveMessageById(message!!.id).complete()
+                                    .retrieveReactionUsers(Emoji.fromUnicode(tutorialEmojis[emo]))) if (!usersReacted.contains(
+                                        user.id
+                                    ) && user.id != bot.selfUser.id
+                                ) {
+                                    usersReacted.add(user.id)
+                                    usersReaction.add(emo)
+                                }
+                            }
+                        } else {
+                            for (emo in projectEmojis.indices.reversed()) {
+                                for (user in bot.getTextChannelById(System.getenv("VOTE_CHANNEL_ID"))!!
+                                    .retrieveMessageById(message!!.id).complete()
+                                    .retrieveReactionUsers(Emoji.fromUnicode(projectEmojis[emo]))) if (!usersReacted.contains(
+                                        user.id
+                                    ) && user.id != bot.selfUser.id
+                                ) {
+                                    usersReacted.add(user.id)
+                                    usersReaction.add(emo)
+                                }
+                            }
+                        }
+                        usersReaction.forEach { num -> sum += num }
+                        val reputation = sum / usersReacted.size
```

Comment:
Maybe move the reputation calculation to a seperate function

Reply:
I don't see a need if it is only two lines, as everything above is required